ui/webui/test/check-basic

```diff
@@ -44,7 +45,8 @@ class TestBasic(anacondalib.VirtInstallMachineCase):
         # Do not start the installation in non-destructive tests as this performs non revertible changes
         # with the pages basically empty of common elements (as those are provided by the top-level installer widget)
         # we at least iterate over them to check this works as expected
-        i.reach(i.steps.REVIEW)
+        # Create user on the first pass through users step
+        i.reach(i.steps.REVIEW, step_callbacks={i.steps.ACCOUNTS: lambda: create_user(self)})
```

Comment:
I would like that the callback would be by default set if not specified to avoid repeating code.

Reply:
Yes, it is fixed in the next commit.